core/common/src/rpc/IModelReadRpcInterface.ts

```diff
@@ -94,7 +94,7 @@ export abstract class IModelReadRpcInterface extends RpcInterface { // eslint-di
   public async getConnectionProps(_iModelToken: IModelRpcOpenProps): Promise<IModelConnectionProps> { return this.forward(arguments); }
   public async queryRows(_iModelToken: IModelRpcProps, _request: DbQueryRequest): Promise<DbQueryResponse> { return this.forward(arguments); }
   @RpcOperation.allowResponseCaching(RpcResponseCacheControl.Immutable) // eslint-disable-line deprecation/deprecation
-  public async querySubCategories(_iModelToken: IModelRpcProps, _categoryIds: CompressedId64Set): Promise<SubCategoryResultRow[]> { return this.forward(arguments); }
+  public async querySubCategories(_iModelToken: IModelRpcProps, _categoryIds?: CompressedId64Set): Promise<SubCategoryResultRow[]> { return this.forward(arguments); }
```

Comment:
You cannot change a required argument of an RPC interface to become optional, this will break if a newer frontend calls an older backend. First bullet point in last section of [guidelines](https://github.com/iTwin/itwinjs-core/blob/master/core/common/src/rpc/README-RpcInterfaceVersioning.md).

Reply:
I've reverted the change to this existing querySubCategories, and instead added a new rpc method instead.